README.md

```diff
@@ -117,7 +117,10 @@ The config json looks like this (comments are not valid json, but I'm inserting
       "signtool": "/src/signing/venv3/bin/signtool",
 
       // enable debug logging
-      "verbose": true
+      "verbose": true,
+
+      // the path to zipalign. This executable is usually present in $ANDROID_SDK_LOCATION/build-tools/$ANDROID_VERSION/zipalign
+      "zipalign": "/absolute/path/to/zipalign"
```

Comment:
We'll need a puppet patch to install zipalign on the signing-scriptworkers, and likely to add the right path to the `script_config.json`.